packages/w/wofi/package.yml

```diff
@@ -0,0 +1,21 @@
+name       : wofi
+version    : 1.4.1
+release    : 1
+source     :
+    - https://hg.sr.ht/~scoopta/wofi/archive/v1.4.1.tar.gz : e95e35c03551c39178c16ad6213a88e3883a236e942d7f2666c780d934c270bb
+homepage   : https://hg.sr.ht/~scoopta/wofi
+license    : GPL-3.0-or-later
+component  : system.utils
+summary    : Wofi is a rofi-like alternative for wlroots-based wayland compositors
+description: |
+    Wofi is a rofi-like alternative for wlroots-based wayland compositors.
+builddeps  :
+    - pkgconfig(gtk+-3.0)
+    - pkgconfig(wayland-server)
+    - pkg-config
```

Comment:
This package is part of the `system.devel` component which means you do not need to list it. It is already included.